CleverPush/CleverPush/Source/CleverPushInstance.m

```diff
@@ -1675,7 +1680,7 @@ - (void)addSubscriptionTags:(NSArray*)tagIds callback:(void(^)(NSArray *))callba
     });
 }
 
-- (void)removeSubscriptionTags:(NSArray*)tagIds callback:(void(^)(NSArray *))callback{
+- (void)removeSubscriptionTags:(NSArray*)tagIds callback:(void(^)(NSArray <CPSubscription*>*))callback{
```

Comment:
Please rebase this PR again on the latest `master` branch. It includes changes from the other open PR.